rust/main/utils/run-locally/src/solana.rs

```diff
@@ -233,23 +232,44 @@ pub fn start_solana_test_validator(
         .cmd("deploy")
         .arg("environment", SOLANA_ENV_NAME)
         .arg("environments-dir", SOLANA_ENVS_DIR)
-        .arg("built-so-dir", SBF_OUT_PATH)
-        .arg("overhead-config-file", SOLANA_OVERHEAD_CONFIG_FILE);
+        .arg("built-so-dir", SBF_OUT_PATH);
 
+    // Deploy sealeveltest1 core
     sealevel_client_deploy_core
         .clone()
         .arg("local-domain", SOLANA_LOCAL_CHAIN_ID)
-        .arg(
-            "remote-domains",
-            [SOLANA_REMOTE_CHAIN_ID, "9913371", "9913372", "9913373"].join(","),
-        )
         .arg("chain", "sealeveltest1")
         .run()
         .join();
 
+    // Deploy sealeveltest2 core
     sealevel_client_deploy_core
         .arg("local-domain", SOLANA_REMOTE_CHAIN_ID)
-        .arg("remote-domains", SOLANA_LOCAL_CHAIN_ID)
+        .arg("chain", "sealeveltest2")
+        .run()
+        .join();
+
+    const SEALEVETEST1_IGP_PROGRAM_ID: &str = "GwHaw8ewMyzZn9vvrZEnTEAAYpLdkGYs195XWcLDCN4U";
+    const SEALEVETEST2_IGP_PROGRAM_ID: &str = "FArd4tEikwz2fk3MB7S9kC82NGhkgT6f9aXi3C5cw1E5";
+
+    let sealevel_client_deploy_core = sealevel_client
```

Comment:
nit: is `sealevel_client_deploy_core` an accurate name for this var or should it relate to configuring the igp